packages/scss/src/components/inlineMessage/mods.scss

```diff
@@ -1,5 +1,10 @@
+@use '@lucca-front/icons/src/icon/exports' as icon;
+
 @mixin S {
 	--components-inlineMessage-fontSize: var(--sizes-XS-fontSize);
 	--components-inlineMessage-lineHeight: var(--sizes-XS-lineHeight);
-	--components-inlineMessage-icon-fontSize: .75rem;
+
+	.lucca-icon {
+		font-size: 0.75rem;
```

Comment:
isn't it "`@include icon.XXS;`" ?

Reply:
Ah yes, I forget this mixin size was added lately. Fixed.